CONTRIBUTING.md

```diff
@@ -24,34 +29,40 @@ Check out our [README.md] for more details.
 
     * The [InnerSource Commons Slack] [#ispo-working-group] community will post the next meeting.
 
-## New Content
+4. New content.
 
-When contributing new content, it can go into one of a few locations:
+    * Add or improve content in the [Managing InnerSource Projects] book.
+    * If it's a pattern, go ahead and contribute to the [InnerSource Patterns] project.
+    * Contribute training segments to the [InnerSource Learning Path] project.
+    * Add slides, spreadsheets, and other artifacts not easily represented elsewhere to the [ISPO Working Group GDrive]  
+    * Documents that don't fit the previous categories (e.g., graphics, opinionated documents, etc.) can go directly into this `ispo-working-group` repo as markdown.
```

Comment:
I moved this up, but after consolidating the three sections this may be incorrect.

I recommend keeping this repo free of anything except concrete content that guides our working group.  Ephemeral content and images can go in GDrive unless the images help guide our working group.

If you all agree, I'll delete the last bullet.

Reply:
Agree.  Now we have that Managing InnerSource Projects book.  Delete it.